Update the Speedo firmware for v3.0 (#10657)
qmk/qmk_firmware: keyboards/cozykeys/speedo/readme.md

# Speedo

![Speedo](http://assets.cozykeys.xyz/images/keyboards/speedo/speedo-v3.0-angle-led-front_1600x1600.png)

- Keyboard Maintainer: [Paul Ewing](https://github.com/pcewing)
- Hardware Supported: Elite-C
- Hardware Availability: [Speedo Repository](https://github.com/cozykeys/speedo)

Make example for this keyboard (after setting up your build environment):

    make speedo:default

> Example of flashing this keyboard:
> 
>     make speedo:default:dfu

Reviewer: ```suggestion
Flashing example for this keyboard:

    make speedo:default:flash
```
